src/style.css

```diff
@@ -0,0 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.formBar {
+  margin-top: 20px;
+  position: relative;
+}
+
+.searchBar {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  line-height: 22px;
+  cursor: zoom-in;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1) ;
+  border-radius: 4px;
+}
+
+.searchBar:hover {
+  font-family: inherit;
+  background: #fff;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar:focus {
+  font-family: inherit;
+  background: linear-gradient(180deg, #fff 0%,#f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.big--searchBar {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+}
+
+.big--searchBar::placeholder {
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.small--searchBar {
+  font-family: inherit;
+  font-weight: 300;
+  padding-left: 33px;
+  height: 42px;
+  font-size: 14px;
+}
+
+.small--searchBar::placeholder {
+  font-family: inherit;
+  color: #3d4e61;
+}
```

Comment:
Follow the checklist, please. Don't repeat the same styles 
<img width="999" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/afa880c6-90fa-4c4c-92cd-026ec379af5c">

Reply:
Hi, just one clarification on this one. So you mean i need to create only one main class and then a separate class only for the small one? For example one class="searchbar" and then only another one class="small--searchbar" and put the styles accordingly? Does the same go for the icon classes i created? * Can you please give me a hint if i only need to use outline:none and font weight: 800; only on the focus input? , Sorry i didn't have the option to comment on the first suggested change*.

Reply:
@Anastasiia-Svintsova